models/sessions/snowplow_mobile_sessions.sql

```diff
@@ -10,7 +10,10 @@
       "data_type": "timestamp"
     },
     cluster_by=snowplow_mobile.mobile_cluster_by_fields_sessions(),
-    tags=["derived"]
+    tags=["derived"],
+    post_hook="{{ snowplow_mobile.stitch_user_identifiers(
```

Comment:
I don't think this will work as there is currently no `stitched_user_id` column in the sessions table.

Reply:
Apologies if I have jumped the gun and you were planning on adding this in a subsequent commit :)